src/test/resources/overbroadConstraintsTest/suggest_double_star

```diff
@@ -0,0 +1,9 @@
+versions.props:
+com.*:* = 1.0.0
+
+versions.lock:
+com.example.core:module:1.0.0 (2 constraints: abcdef1)
+com.different.example:artifact:2.0.0 (2 constraints: abcdef1)
+
+new pins (blank when no pins needed):
+com.different.*:* = 2.0.0
```

Comment:
I'm not sure if we ever want to suggest a star in a group, as we really can't be sure if this is right (it's almost certainly not for the software we produce at least). I think perhaps only adding stars in the artifact position makes sense/is enough.